<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AlashPars</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../media/main_rs.qrc">
    <normaloff>:/logo/parsing.ico</normaloff>:/logo/parsing.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>980</width>
         <height>540</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QTabWidget" name="homeTab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Cambria</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::West</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="simulationTab">
           <attribute name="icon">
            <iconset resource="../media/main_rs.qrc">
             <normaloff>:/logo/internet.ico</normaloff>:/logo/internet.ico</iconset>
           </attribute>
           <attribute name="title">
            <string>Парсинг</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="3" column="0" colspan="4">
             <widget class="QTableWidget" name="activityMonitor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(217, 236, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
              <property name="sortingEnabled">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>200</number>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>30</number>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Время исполнения </string>
               </property>
               <property name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/time.ico</normaloff>:/logo/time.ico</iconset>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Сообщение</string>
               </property>
               <property name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/comment.ico</normaloff>:/logo/comment.ico</iconset>
               </property>
              </column>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QProgressBar" name="progressBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStatesMinorOutlyingIslands"/>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>1</number>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="textDirection">
               <enum>QProgressBar::TopToBottom</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="clearMessageTableButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Очистить</string>
              </property>
              <property name="icon">
               <iconset resource="../media/main_rs.qrc">
                <normaloff>:/logo/clear.ico</normaloff>:/logo/clear.ico</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="simulationDownloadProgress">
              <property name="text">
               <string>Процесс обработки</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="simulationInterfaceGroup">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Команды</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QPushButton" name="configureButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Настройки</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../media/main_rs.qrc">
                   <normaloff>:/logo/settings2.ico</normaloff>:/logo/settings2.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="runButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Запуск</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../media/main_rs.qrc">
                   <normaloff>:/logo/run.ico</normaloff>:/logo/run.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="endButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Прервать</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../media/main_rs.qrc">
                   <normaloff>:/logo/stop.ico</normaloff>:/logo/stop.ico</iconset>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="newsTab">
           <attribute name="icon">
            <iconset resource="../media/main_rs.qrc">
             <normaloff>:/logo/base_data.ico</normaloff>:/logo/base_data.ico</iconset>
           </attribute>
           <attribute name="title">
            <string>База</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0">
             <widget class="QTabWidget" name="tabWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_7">
               <attribute name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/perm_table.ico</normaloff>:/logo/perm_table.ico</iconset>
               </attribute>
               <attribute name="title">
                <string>Постоянная таблица</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="1">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Фильтр</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="8">
                 <widget class="QTableView" name="permanent_tableView">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(217, 236, 255);</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="7">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>679</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <widget class="QComboBox" name="filter_comboBox">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>Все</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Товары с данными</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Товары без данных</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_8">
               <attribute name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/temp_table.ico</normaloff>:/logo/temp_table.ico</iconset>
               </attribute>
               <attribute name="title">
                <string extracomment="Для хранения промежуточных данных">Временная таблица</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QTableView" name="temporary_tableView">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3"/>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Поиск по атрибуту:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="search_table_articul_lineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_update">
                <property name="text">
                 <string>Обновить</string>
                </property>
                <property name="icon">
                 <iconset resource="../media/main_rs.qrc">
                  <normaloff>:/logo/update.ico</normaloff>:/logo/update.ico</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="icon">
            <iconset resource="../media/main_rs.qrc">
             <normaloff>:/logo/help (1).ico</normaloff>:/logo/help (1).ico</iconset>
           </attribute>
           <attribute name="title">
            <string>Помощь</string>
           </attribute>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>160</width>
              <height>86</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButton_update_app">
               <property name="text">
                <string>Обновление</string>
               </property>
               <property name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/update.ico</normaloff>:/logo/update.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_about_app">
               <property name="text">
                <string>О программе</string>
               </property>
               <property name="icon">
                <iconset resource="../media/main_rs.qrc">
                 <normaloff>:/logo/about.ico</normaloff>:/logo/about.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="configurationAction">
   <property name="text">
    <string>Bot Settings</string>
   </property>
  </action>
  <action name="otherCommandsAction">
   <property name="text">
    <string>Miscellaneous</string>
   </property>
  </action>
  <action name="statisticsAction">
   <property name="text">
    <string>Statistics</string>
   </property>
  </action>
  <action name="aboutAlgobotAction">
   <property name="text">
    <string>About Algobot</string>
   </property>
  </action>
  <action name="liveStatisticsAction">
   <property name="text">
    <string>Live</string>
   </property>
  </action>
  <action name="simulationStatisticsAction">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="openLogFolderAction">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="openBacktestResultsFolderAction">
   <property name="text">
    <string>Backtest Results</string>
   </property>
  </action>
  <action name="openCsvFolderAction">
   <property name="text">
    <string>CSVs</string>
   </property>
  </action>
  <action name="sourceCodeAction">
   <property name="text">
    <string>Source Code</string>
   </property>
  </action>
  <action name="openDatabasesFolderAction">
   <property name="text">
    <string>Databases</string>
   </property>
  </action>
  <action name="openConfigurationsFolderAction">
   <property name="text">
    <string>Configuration Files</string>
   </property>
  </action>
  <action name="openCredentialsFolderAction">
   <property name="text">
    <string>Credential Files</string>
   </property>
  </action>
  <action name="tradingViewLiveAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </action>
  <action name="tradingViewSimulationAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="tradingViewBacktestAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backtest</string>
   </property>
  </action>
  <action name="binanceLiveAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Live</string>
   </property>
  </action>
  <action name="binanceSimulationAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="binanceBacktestAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backtest</string>
   </property>
  </action>
  <action name="binanceHomepageAction">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="tradingViewHomepageAction">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="openOptimizerResultsFolderAction">
   <property name="text">
    <string>Optimizer Results</string>
   </property>
  </action>
  <action name="openVolatilityResultsFolderAction">
   <property name="text">
    <string>Volatility Results</string>
   </property>
  </action>
  <action name="wikiAction">
   <property name="text">
    <string>Wiki</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../media/main_rs.qrc"/>
 </resources>
 <connections/>
</ui>
